src/ExpressServer.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express';
+import { Server as HttpServer } from 'http';
+import { register } from 'prom-client';
+import { ErrorHandler } from './ErrorHandler';
+import { Logger } from './Logger';
+
+export class ExpressServer {
+    private expressServer: express;
+    private serverPort: number;
+    private server: HttpServer;
+
+    constructor(serverPort: number, private readonly logger: Logger) {
+        this.serverPort = serverPort;
+        this.expressServer = express();
+        this.initRoutes();
+        // Error handler
+        this.expressServer.use(ErrorHandler(this.logger));
+    }
+
+    private initRoutes(): void {
+        this.expressServer.get('/metrics', async (req, res) => {
+            res.set('Content-Type', register.contentType);
+            res.end(register.metrics());
+        });
+    }
+
+    start(): void {
+        this.server = this.expressServer.listen(this.serverPort, () => {
+            this.logger.info(`App server listening on port ${this.serverPort}!`);
+        });
+    }
+
+    close(cb: () => any): void {
```

Comment:
Plus `void` que `any` sur le retour du callback

```suggestion
    close(cb: () => void): void {
```